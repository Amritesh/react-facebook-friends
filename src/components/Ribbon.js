import React from 'react';
import cx from 'classnames';
import '../style/Ribbon.css';

const Ribbon = () => (
  <div className={cx('github-fork-ribbon-wrapper', 'left', 'fixed', 'ribbon-wrapper')}>
    <div className="github-fork-ribbon">
      <a target="_blank" rel="noopener noreferrer" href="https://github.com/LeoAJ">Fork me on Github</a>
    </div>
  </div>
);

export default Ribbon;
