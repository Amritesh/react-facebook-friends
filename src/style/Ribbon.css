.ribbon-wrapper {
  display: block;
}

@media (max-width: 48em) {
  .ribbon-wrapper {
    display: none;
  }
}