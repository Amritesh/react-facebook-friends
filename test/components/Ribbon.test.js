import React from 'react';
import renderer from 'react-test-renderer';
import Ribbon from '../../src/components/Ribbon';

test('Ribbon component should renders correctly', () => {
  const component = renderer.create(<Ribbon />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
