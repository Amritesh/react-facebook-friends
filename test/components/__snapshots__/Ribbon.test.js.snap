// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ribbon component should renders correctly 1`] = `
<div
  className="github-fork-ribbon-wrapper left fixed ribbon-wrapper"
>
  <div
    className="github-fork-ribbon"
  >
    <a
      href="https://github.com/LeoAJ"
      rel="noopener noreferrer"
      target="_blank"
    >
      Fork me on Github
    </a>
  </div>
</div>
`;
